### PR TITLE
Update dependencies and solve warnings

### PR DESCRIPTION
**Pull request by adolfosilva**
This PR does the following:
 - code formatting with `mix format`
 - remove unused :earmark dependency
 - updates remaining dependencies
 - solve warnings when running with Elixir 1.10.3
 - bumps version to 0.2.1

Any feedback is welcome. Happy to squash and merge. Thanks!

### CODE REVIEW

File: .formatter.exs [hunk 1]
```diff
@@ -0,0 +1,4 @@
+# Used by "mix format"
```
Comment (cwbriones) [inline .formatter.exs]: Can you remove this comment?

**Review by cwbriones — COMMENTED** [review]
Thanks for the cleanup!

Other than the (very minor) comment I've left, can you also bump the version number in the README installation instructions to `0.2.1`?

File: .formatter.exs (hunk 1 above)
Comment (adolfosilva) [reply .formatter.exs]: Done

**Comment by adolfosilva** [comment]
> Thanks for the cleanup!
> 
> Other than the (very minor) comment I've left, can you also bump the version number in the README installation instructions to `0.2.1`?

The version bump is done in https://github.com/cwbriones/scientist/pull/4/commits/99c11b3cd37bd60a1f3e9985ac187911f7ef2476

**Review by cwbriones — APPROVED** [review]
> The version bump is done in 99c11b3

And indeed it is!

**Pull request merged**